express/features/sign-in.feature

```diff
@@ -0,0 +1,161 @@
+Feature: Happy paths for the self-service sign-in flow
+
+  Background:
+    Given that the user is on the "/sign-in" page
+
+  Scenario: User does not enter a valid email address
+    When the user submits the email "invalid-email.com"
+    Then the error message "Enter an email address in the correct format, like name@example.com" must be displayed for the "emailAddress" field
+
+  Scenario: The user does not enter any characters into the free text field
+    When the user submits the email ""
+    Then the error message "Enter your email address" must be displayed for the "emailAddress" field
+
+  Scenario: The user’s email address is saved with the service
+    When they enter "registered@gds.gov.uk" into the "emailAddress" field
+    When they click the Continue button
+    Then they should be redirected to the "/sign-in-password" page
+
+  Scenario: User doesn’t enter any characters
```

Comment:
```suggestion
  Scenario: User doesn’t enter any characters into the password field
```

Reply:
Yes, It make more sense. I agree, I was following the AC done by Rohela. Need to update user story for that. I can do.